manage.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.docker_settings")
```

Comment:
Just a standard setting file should be okay.

You can specify docker-compose specific environments by using https://docs.docker.com/compose/environment-variables/#set-environment-variables-in-containers